discord-bot/src/main/java/fr/pacifista/bot/discord/events/MessageInteractions.java

```diff
@@ -0,0 +1,72 @@
+package fr.pacifista.bot.discord.events;
+
+import fr.pacifista.api.support.tickets.client.dtos.PacifistaSupportTicketDTO;
+import fr.pacifista.api.support.tickets.client.dtos.PacifistaSupportTicketMessageDTO;
+import fr.pacifista.bot.discord.api.PacifistaTicketClient;
+import fr.pacifista.bot.discord.api.PacifistaTicketMessageClient;
+import fr.pacifista.bot.discord.config.Config;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.channel.Channel;
+import net.dv8tion.jda.api.entities.channel.ChannelType;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import net.dv8tion.jda.api.events.message.MessageUpdateEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Date;
+
+public class MessageInteractions extends ListenerAdapter {
```

Comment:
Essayer de déclarer cet event en @Service en injectant les clients feign (si tu n'y arrive pas je le ferais vu que j'ai pas encore implémenté la communication vers l'api avec le bot)

Reply:
C'est un évent qui doit être ajouté au JDA en créant une instance de la classe, je pense pas comprendre assez bien Spring pour l'instant pour utiliser parfaitement ca

Reply:
Tu peux injecter JDA et ajouter la classe event dans le constructeur de la classe avec un "this"